proto/firecracker.proto

```diff
@@ -32,6 +32,11 @@ message CreateVMRequest {
 
     // Whether the VM should exit after all tasks running in it have been deleted.
     bool ExitAfterAllTasksDeleted = 9;
+
+    // Determines whether or not the jailer should be disabled.
+		//
+		// Valid values are "ON", "OFF"
+		string Jailing = 10;
```

Comment:
I think it might be more flexible in the long-run to make a separate struct for `JailingConfig` or something. Right now it may just have a single bool field to enable/disable, but doing that would allow us to add more fields to it in the future in a more backwards-compatible way.